vsphere/internal/helper/spbm/spbm_helper.go

```diff
@@ -0,0 +1,142 @@
+package spbm
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/terraform-providers/terraform-provider-vsphere/vsphere/internal/helper/provider"
+	"github.com/terraform-providers/terraform-provider-vsphere/vsphere/internal/helper/viapi"
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/pbm"
+	"github.com/vmware/govmomi/pbm/methods"
+	pbmtypes "github.com/vmware/govmomi/pbm/types"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+// pbmClientFromGovmomiClient creates a new pbm client from given govmomi client.
+// Can we have it in govmomi client as a field similar to tag client?
+// We should not create a new pbm client every time we need it.
+func pbmClientFromGovmomiClient(ctx context.Context, client *govmomi.Client) (*pbm.Client, error) {
```

Comment:
I agree this would be better create the pbm client as part of the global config. Given the current structure, I think its ok to leave here for now. There is another feature in progress that needs some modification to the client structure, so I'll see about including the pbm client in that work.

Reply:
Just merged a [PR](https://github.com/terraform-providers/terraform-provider-vsphere/blob/master/vsphere/config.go#L37) that adds the pbm client to the vSphereClient.

Reply:
Is there a way to get vSphereClient from govmomi.Client?
We have govmomi.Client handle in VM operations and need pbm.Client from it. The vSphereClient struct will have pbm.Client.